plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/jdbc/TestingIcebergJdbcServer.java

```diff
@@ -64,12 +64,10 @@ public void execute(@Language("SQL") String sql)
     public String getJdbcUrl()
     {
         return format(
-                "jdbc:postgresql://%s:%s/%s?user=%s&password=%s&currentSchema=tpch,public",
+                "jdbc:postgresql://%s:%s/%s?currentSchema=tpch,public",
```

Comment:
Why do we need `currentSchema=tpch,public` ?

Same question for `TestingJdbcServer` where after removing this string from the connection URL the tests were still 🟢

Reply:
btw, this is also appearing in the product test environment configuration.